src/index.html

```diff
@@ -136,6 +136,11 @@
                   >#sponsors</a
                 >
               </li>
+              <li class="main-nav__item">
+                <a class="main-nav__link" href="#tickets" title="Tickets" aria-label="Tickets"
+                  >#tickets</a
```

Comment:
![Screenshot 2022-07-02 at 18-02-49 meet js Summit 2019](https://user-images.githubusercontent.com/509375/177007879-e525a20c-a63c-4740-b09e-d37830e16507.png)

tickets link used to be a button